src/Components/Error.jsx

```diff
@@ -0,0 +1,13 @@
+const Error = ({ error }) => {
+  return (
+    <div className="error-display">
+      <h2>Whoops! There seems to be a problem</h2>
+      <p>An error occured while trying to fetch data:</p>
+      <pre>
+        {error.status} {error.response.data.msg}
+      </pre>
+    </div>
+  );
+};
```

Comment:
You want to be careful here how much technical information you are showing to a user about an error, this really comes down to how technically worded your `error.response.data.msg` strings are from your API

Reply:
just remember than in frontend we want to keep things user friendly as most users wouldn't be developers